include/gz/sim/rendering/MarkerManager.hh

```diff
@@ -20,6 +20,7 @@
 #include <memory>
 #include <string>
 
+#include <gz/sim/config.hh>
```

Comment:
Curious why this was needed? Or did your IDE add it.

Reply:
It's needed to successfully link. The `GNITION_GAZEBO_VERSION_NAMESPACE` is undefined unless this is included. I think no one had try to directly utilize this file, other than RenderUtil.